testutil/lowlevel.go

```diff
@@ -218,16 +228,35 @@ func (sys *SyscallRecorder) InsertFaultFunc(call string, fn func() error) {
 
 // Calls returns the sequence of mocked calls that have been made.
 func (sys *SyscallRecorder) Calls() []string {
```

Comment:
Do we still need this now that we have "RCalls" which looks like a super-set?

Reply:
Unless dropping it will inflate the PR significantly, then I guess a follow-up would be ok.

Reply:
Yes because existing code :) But subsequent branches will remove it.